src/SAWScript/Crucible/JVM/Builtins.hs

```diff
@@ -284,7 +284,7 @@ jvm_unsafe_assume_spec cls nm setup =
      pos <- getPosition
      -- cls' is either cls or a (transitive) superclass of cls
      (cls', method) <- io $ findMethod cb pos nm cls -- TODO: switch to crucible-jvm version
-     let loc = SS.toW4Loc "_SAW_assume_spec" pos
+     let loc = SS.toW4Loc "_SAW_JVM_unsafe_assume_spec" pos
```

Comment:
...but this one (and the ones below it) use `_SAW_JVM_*`, where `JVM` is uppercase. We should pick one convention and use it throughout.

(Similar comments apply to `_SAW_mir_*` and `_SAW_MIR_*` elsewhere.)